### PR TITLE
[Refactor] Clean Master Code and Update README

### PR DESCRIPTION
**Pull request by liuruiyiyang**
Change List:
1. Update the README docs
2. Remove the old ```db``` package 
3. Add MySQL Log for the new mixed workload mode.
4. Add Basic Tableau Visualization WorkBook for the new mixed workload mode.

Things to be done:
1. Add OpenTSDB into the new benchmark framework.
2. Perfect the MySQL Log（Recorder）of the new benchmark framework.

### CODE REVIEW

File: conf/config.properties [hunk 1]
```diff
@@ -87,9 +88,6 @@ COMPRESSOR=SNAPPY
 CREATE_SCHEMA=true
 #数据库初始化等待时间ms
 INIT_WAIT_TIME=5000
-#若运行模式为insertTestWithUserDefinedPath则表示生成样本数据SQL的文件路径，若路径不存在，视为不需要写入文件，否则自动新建一个名为sampleData.txt的文件并写入SQL语句
-#若为testWithDefaultPath则表示DB engine生成数据所在路径
-GEN_DATA_FILE_PATH=/home/liurui
 #DB所在服务器中data文件和log_stop_flag所在路径
 LOG_STOP_FLAG_PATH=/home/ubuntu/first-rest-test/monitor/iotdb-benchmark
```
[the remark this answers is not in the document]
Comment (liuruiyiyang) [reply conf/config.properties]: I rename LOG_STOP_FLAG_PATH to DB_DATA_PATH and add some descriptions

File: conf/config.properties [hunk 2]
```diff
@@ -19,40 +19,41 @@ DB_URL=http://192.168.8.2:6666
...
+#Q5--带值过滤的聚合查询 select func(v1)... from data where device in ? and value > ?
+#Q6--带值过滤和时间过滤的聚合查询 select func(v1)... from data where device in ? and value > ? and time > ? and time < ?
+#Q7--分组聚合查询（GROUP BY）（暂时只能生成有一个时间区间的语句）
+#Q8--最近点查询 select time, v1... where device = ? and time = max(time)
 OPERATION_PROPORTION=2:0:0:0:0:0:0:0:0
-#启动测试前是否清除旧数据(是否执行cleanup)
+#启动测试前是否清除旧数据(是否执行cleanup) 对IoTDB除外 因为IoTDB不支持删除Storage Group
 IS_DELETE_DATA=false
 #是否使用prepareStatement,如果为false,需要指定USE_SESSION属性
 USE_PREPARE_STATEMENT=false
```
[the remark this answers is not in the document]
Comment (liuruiyiyang) [reply conf/config.properties]: deleted

File: README.md [hunk 3]
```diff
@@ -24,56 +22,55 @@ IoTDB-benchmark's features are as following:
...
+HOST=127.0.0.1
 PORT=6667
 DB_SWITCH=IoTDB
-BENCHMARK_WORK_MODE=insertTestWithDefaultPath
-GROUP_NUMBER=10
-DEVICE_NUMBER=100
-SENSOR_NUMBER=100
-CLIENT_NUMBER=10
+BENCHMARK_WORK_MODE=testWithDefaultPath
+OPERATION_PROPORTION=2:0:0:0:0:0:0:0:0
```
Comment (EJTTianYu) [inline README.md]: any meaning for write percentage is 2？

File: README.md [hunk 4]
```diff
@@ -82,53 +79,202 @@ LOOP=1000
...
+INGESTION               0               0               0               0               0.00            0.00            
+PRECISE_POINT           26              26              0               0               61.04           413.49          
+TIME_RANGE              27              1377            0               0               3232.57         16822.62                
+VALUE_RANGE             18              918             0               0               2155.05         17844.87                
+AGG_RANGE               16              16              0               0               37.56           314.56          
+AGG_VALUE               16              16              0               0               37.56           267.43          
+AGG_RANGE_VALUE         46              46              0               0               107.99          416.17          
+GROUP_BY                15              195             0               0               457.77          9267.95         
+LATEST_POINT            36              36              0               0               84.51           482.57
```
Comment (EJTTianYu) [inline README.md]: the query result may be not consistent with the config above.

File: conf/config.properties [hunk 5]
```diff
@@ -19,40 +19,39 @@ DB_URL=http://192.168.8.2:6666
...
+#Q2--范围查询（只限制起止时间）select v1... from data where time > ? and time < ? and device in ?
+#Q3--带值过滤的范围查询 select v1... from data where time > ? and time < ? and v1 > ? and device in ?
+#Q4--带时间过滤的聚合查询 select func(v1)... from data where device in ? and time > ? and time < ?
+#Q5--带值过滤的聚合查询 select func(v1)... from data where device in ? and value > ?
+#Q6--带值过滤和时间过滤的聚合查询 select func(v1)... from data where device in ? and value > ? and time > ? and time < ?
+#Q7--分组聚合查询（GROUP BY）（暂时只能生成有一个时间区间的语句）
+#Q8--最近点查询 select time, v1... where device = ? and time = max(time)
 OPERATION_PROPORTION=2:0:0:0:0:0:0:0:0
-#启动测试前是否清除旧数据(是否执行cleanup)
+#启动测试前是否清除旧数据(是否执行cleanup) 对IoTDB除外 因为IoTDB不支持删除Storage Group
```
Comment (EJTTianYu) [inline conf/config.properties]: now it seems that IoTDB has delete SG function using session. However, we can not delete SG using jdbc.

File: src/main/java/cn/edu/tsinghua/iotdb/benchmark/App.java [hunk 6]
```diff
@@ -280,12 +246,9 @@ private static void finalMeasure(ExecutorService executorService, CountDownLatch
     /**
      * 测试真实数据集
      *
-     * @param config
+     * @param config configurations
```
Comment (EJTTianYu) [inline src/main/java/cn/edu/tsinghua/iotdb/benchmark/App.java]: maybe here use "Config instance" is better.

**Pull request merged**